types/culori/test/average.test.ts

```diff
@@ -1,31 +0,0 @@
-import { average } from "culori";
-
-// @ts-expect-error
-average();
```

Comment:
cc @bijela-gora no tests were deleted; the DefinitelyTyped project just seemed to prefer moving to one `culori-tests.ts` file.

Reply:
I feel like it's better to have tests in separate files. WDYT?

In the [DT README](https://github.com/DefinitelyTyped/DefinitelyTyped?tab=readme-ov-file#my-package-teststs), we have:
> There should be a <my-package>-tests.ts file, which is considered your test file, **along with any \*.ts files it imports.**

So, we can import test files inside `culori-tests.ts` like this:
![image](https://github.com/user-attachments/assets/b156039d-d335-47f1-bdab-747aee13b9f9)

Reply:
I dug in a little more and actually the `[package]-tests.ts` format doesn’t matter! The only thing is just adding to `tsconfig.json`. Restored all the previous tests to their original locations